Fabric.Identity.API/scripts/tests/Install-IdPSS-Utilities.tests.ps1

```diff
@@ -395,5 +395,89 @@ Describe 'IdPSS Unit Tests' {
         }
     }
 }
+Describe "IdPSS Integration Tests"{
+    Describe 'Register-Identity' -Tag 'Integration'{
+        BeforeEach{
+            # Arrange 
+            # Add to the powershell TestDrive which cleans up after each context, leaving the tests folder configs unchanged
+            
+            $Global:testAzureFile = "testAzure.config"
+            $Global:testAzureFileLoc = "$PSScriptRoot\testAzure.config"
+            $Global:azureConfigPath = "$($TestDrive)\$($testAzureFile)"
+            $doesAzureFileExist = Test-Path $azureConfigPath
+            if (!$doesAzureFileExist)
+            {
+                Get-Content "$testAzureFileLoc" | Out-File $azureConfigPath
+            }
+        }
+        AfterEach{
+            # test file will exist within the same context, so it needs to be blown away
+            $doesAzureFileExist = Test-Path $azureConfigPath
+            if ($doesAzureFileExist)
+            {
+                Remove-Item $azureConfigPath
+            }
+        } 
+        Context 'Register Identity With Azure SecretName should work : PBI 195930'{
+            InModuleScope Install-IdPSS-Utilities{
+                It 'Should successfully register the identity'{
+                    Mock -CommandName Write-Host {}
+                    Mock -CommandName Connect-AzureADTenant {}
+                    Mock -CommandName Get-GraphiApiUserDirectoryReadPermissions{}
+                    Mock -CommandName New-FabricAzureADApplication { return @{ AppId = "AppId"; ObjectId = "ObjectId"}}
+                    Mock -CommandName Get-FabricAzureADSecret { return "secret"}
+                    Mock -CommandName Disconnect-AzureAD {}
+                    Mock -CommandName Add-InstallationTenantSettings {}
+                    Mock -CommandName Start-Process {}
+
+                    # Act
+                    Register-Identity -appName "Test Name" -replyUrls @{} -configSection "identity" -azureConfigPath $azureConfigPath 
```

Comment:
I guess there isn't anything we can # Assert, since this function doesn't return anything and an empty string for secretname results in a failure inside function Confirm-InstallIdpSSUtilsSecretName.